firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -280,3 +304,156 @@ uint8_t TCMDEXEC_available_telecommands(const char *args_str, TCMD_TelecommandCh
 
     return 0;
 }
+
+uint8_t TCMDEXEC_set_int_config_variable(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
+                                         char *response_output_buf, uint16_t response_output_buf_len)
+{
+    const int args_str_len = strlen(args_str);
+    char config_var_name[50];
+    memset(config_var_name, 0, 50);
+    const uint8_t parse_result = TCMD_extract_string_arg(args_str, 0,config_var_name, 50);
+    if (parse_result > 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "%s : %s\n", "Could not parse index 0 for: ", args_str);
+        return 0;
+    }
+
+    uint64_t config_var_new_value;
+    const uint8_t parse_result2 = TCMD_extract_uint64_arg(args_str,args_str_len , 1, &config_var_new_value);
+    if (parse_result2 > 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "%s : %s\n", "Could not parse index 1 for: ", args_str);
+        return 0;
+    }
+
+    CONFIG_integer_config_entry_t *res;
+    const uint8_t found_int = CONFIG_find_integer_variable_by_name((const char *)config_var_name, strlen((char *)config_var_name), &res);
+    if (found_int > 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "%s : %s\n", "Could not find config var: ", config_var_name);
+        return 0;
+    }
+
+    const uint8_t set_result = CONFIG_assign_int_new_value(&res, config_var_new_value);
+
+    if (set_result > 0)
+    {
+        snprintf(response_output_buf, response_output_buf_len, "Could not set config var : %s to %lu\n", config_var_name, (uint32_t)config_var_new_value);
```

Comment:
The only reason the above could fail is because the value is too large for a `uint32_t`. You can't go and type cast it to a uint32_t and display it to the user, expecting it to do anything other than confuse the satellite operator. 

Options: use the function in `arrays.h` to convert to string, or remove the printout of `config_var_new_value`

Reply:
I had trouble using the 64bit format specifier (%llu) as it would not print the number, but the string 'lu'. Because I defaulted all numbers to be uint32, I just cast it to 32 bit (safe because we are casting from a larger data type to a smaller data type) so I could print using the %lu format specifier, which works